pkgs/servers/frr/libyang.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pcre
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libyang";
+  version = "1.0.225";
+  src = fetchFromGitHub {
+    owner = "CESNET";
+    repo = "libyang";
+    rev = "v${version}";
+    sha256 = "1f00j7jyszcz9zx41w3amm54sk6qq5m5lzkdkxg79pl1n1r69yr4";
+  };
+  nativeBuildInputs = [ cmake pkg-config ];
+  buildInputs = [ pcre ];
+  cmakeFlags = [ "-DENABLE_LYD_PRIV=ON" "-DCMAKE_BUILD_TYPE:String=Release" ];
+}
```

Comment:
Missing meta. Why is this not a top-level entry?

Reply:
Just was not sure if it is worth to be a top-level entry.

Reply:
where to put it in the tree? `pkgs/development/tools/`?

Reply:
libraries, of course. done.